test/irc_test.rb

```diff
@@ -291,6 +291,37 @@ def test_no_colors
     refute_match /\003/, privmsg
   end
 
+  def test_public_repo_format_in_irc_realname
+    svc = service({'room' => 'r', 'nick' => 'n'}, payload)
+
+    svc.receive_push
+    msgs = svc.writable_irc.string.split("\n")
+
+    assert_includes msgs, "USER n 8 * :GitHub IRCBot - mojombo/grit"
+  end
+
+  def test_private_repo_format_in_irc_realname
+    payload_copy = payload.clone
```

Comment:
You don't need to make a clone of the payload. Each call to `#payload` returns a new object. See:

- https://github.com/github/github-services/blob/605d76c/test/helper.rb#L37-L40
- https://github.com/github/github-services/blob/2e886f4/lib/service/events/push_helpers.rb#L171-L226.

@Zarthus: Can I talk you into removing the `#clone` invocation from this test and the one below?